requirements.txt

```diff
@@ -5,4 +5,5 @@ matplotlib
 scikit-image
 future
 colour-science
-colour-checker-detection
\ No newline at end of file
+colour-checker-detection
+ipympl
```

Comment:
I was not 100% sure if we wanted to require this, but I guess it is not too big, and it probably makes sense.